### PR TITLE
some cleanup

### PR DESCRIPTION
**Pull request by radioegor146**

### CODE REVIEW

File: PassGen/MainWindow.xaml.cs [hunk 1]
```diff
@@ -1,74 +1,74 @@
...
+            }
+            ResultTextBox.Text = resultPassword;
         }
 
-        private void PLength_ValueChanged(object sender, RoutedPropertyChangedEventArgs<double> e)
+        private void PasswordLengthSliderValueChanged(object sender, RoutedPropertyChangedEventArgs<double> e)
         {
-            Current.Content = PLength.Value;
+            if (CurrentPasswordLengthLabel != null)
+                CurrentPasswordLengthLabel.Content = PasswordLengthSlider.Value;
```
Comment (DCNick3) [inline PassGen/MainWindow.xaml.cs:66]: Why not use `variable?` syntax for checking for null?

File: PassGen/MainWindow.xaml.cs [hunk 2]
```diff
@@ -1,74 +1,74 @@
...
+        private void GenerateButtonClick(object sender, RoutedEventArgs e)
         {
-            string s = String.Empty;
-            for (int i = 0; i < PLength.Value; i++)
+            // for more info check ASCII table
+
+            var resultPassword = string.Empty;
+
+            const int startShift = ' ' + 1;
+            const int charRange = 'z' - '!' + 1;
```
Comment (DCNick3) [inline PassGen/MainWindow.xaml.cs:30]: Why not write `'!'` in `startShift` (as it's more obvious)?

File: PassGen/MainWindow.xaml.cs [hunk 3]
```diff
@@ -1,74 +1,74 @@
...
+                if (UseDigitsCheckBox.IsChecked == true)
                 {
-                    if (Symbols.IsChecked == false)
+                    if (UseSpecialSymbolsCheckBox.IsChecked == false)
                     {
-                        while (xchar < 48 || xchar > 57 && xchar < 65 || xchar > 90 && xchar < 97 || xchar > 122)
-                        {
-                            xchar = 33 + getNewNumber(94);
-                        }
+                        while (charValue < '0' || charValue > '9' && charValue < 'a' || charValue > 'z' && charValue < 'A' || charValue > 'Z')
```
Comment (DCNick3) [inline PassGen/MainWindow.xaml.cs:40]: Broken loop, as `'a'` is bigger than `'A'`
Should be `charValue < '0' || charValue > '9' && charValue < 'A' || charValue > 'Z' && charValue < 'a' || charValue > 'z'`

File: PassGen/MainWindow.xaml.cs [hunk 4]
```diff
@@ -1,74 +1,74 @@
...
                 else
                 {
-                    if (Symbols.IsChecked == false)
+                    if (UseSpecialSymbolsCheckBox.IsChecked == false)
                     {
-                        while (xchar < 65 || xchar > 90 && xchar < 97 || xchar > 122)
-                        {
-                            xchar = 33 + getNewNumber(94);
-                        }
+                        while (charValue < 'a' || charValue > 'z' && charValue < 'A' || charValue > 'Z')
```
Comment (DCNick3) [inline PassGen/MainWindow.xaml.cs:48]: Same as above

File: PassGen/MainWindow.xaml.cs (hunk 1 above)
Comment (radioegor146) [reply PassGen/MainWindow.xaml.cs:66]: How can you use it here? Provide example :)

Comment (DCNick3) [reply PassGen/MainWindow.xaml.cs:66]:
```
CurrentPasswordLengthLabel?.Content = PasswordLengthSlider.Value;
```

Comment (radioegor146) [reply PassGen/MainWindow.xaml.cs:66]: :thinking:

Comment (DCNick3) [reply PassGen/MainWindow.xaml.cs:66]: Well, this is weird

**Pull request merged**